src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const parts = sourceString.split(';');
+  const result = {};
```

Comment:
The variable naming `result` is not descriptive enough. Consider using a name that reflects the data it holds, such as `styleObject` or `cssProperties`.